apps/labs/types/storyblok/bloks/posts.ts

```diff
@@ -29,7 +29,7 @@ export type TPostMeta = {
   title: string;
   published: string;
   description?: string;
-  author: TPostAuthor;
+  author: TPostAuthor[];
```

Comment:
Let's make this attribute plural:

```suggestion
  authors: TPostAuthor[];
```

We'll then have to revise:

- Each blog post's metadata from `author` to `authors`
- The README metadata documentation, to indicate the attribute name change and also to describe how all authors need to be added as an array.

Reply:
@Surafel-Getachew, please create a first draft of an edit to the README to reflect this metadata attribute and syntax change.